### ISSUE TITLE
Raw strings forced on a separate line by indent rule

### ISSUE THREAD

**Issue by lwasyl** [pinterest/ktlint#711]
KtLint 0.36.0

Raw strings don't seem to play well with indent experimental rule. For example this:

```kotlin
description = """
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris lacus arcu, blandit non semper elementum,
    """.trimIndent(),
```

reports an error: `[experimental:indent] Missing newline after "="`

This (weird) formatting:
```kotlin
description =
"""
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris lacus arcu, blandit non semper elementum,
    """.trimIndent(),
```
reports wrong indentation for the first `"""`, however this:
```kotlin
description =
    """
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris lacus arcu, blandit non semper elementum,
    """.trimIndent(),
```
is automatically reformatted by IntellIJ to the previous variant (with first `"""` not indented)

**Comment by henrikbarium** [comment]
This behavior is especially annoying when using single-line raw strings, e.g. as function arguments. (Using named parameters makes it even worse.)

```kotlin
data class Example(val string: String)
val e = Example(string = """a"b"c""")
```

The above code snippet is auto-formatted by ktlint into the following.

```kotlin
data class Example(val string: String)
val e = Example(string =
        """a"b"c""")
```

However, this is not valid according to ktlint, so it also includes some errors.

- Missing newline before ")"
- Missing newline after "("

Running ktlint with auto formatting again then gives the following.

```kotlin
data class Person(val name: String)
val p = Person(
    name =
        """a"b"c"""
)
```

This is quite a lot worse than the initial code example, especially if we add more arguments (which may even contain raw strings).

**Comment by mehwhatever** [comment]
I was planning to work on this, if nobody has already started.

**Comment by romtsn** [comment]
@mehwhatever would be awesome, most likely it's just commenting out these lines:
https://github.com/pinterest/ktlint/blob/94b62e217339bbd6d939e24b41f92ab83e0cb78c/ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/IndentationRule.kt#L336-L339

But I'm not sure, definitely needs some testing.

**Comment by cortinico** [comment]
Adding another example to this issue. The following line:

```kotlin
val html = """<span style="color:red;">my span</span>"""
```

Gets reformatted by ktlint v0.37.2 to this:

```kotlin
val html =
    """<span style="color:red;">my span</span>"""
```
